lab_1_classify_by_unigrams/main.py

```diff
@@ -11,14 +11,33 @@ def tokenize(text: str) -> list[str] | None:
     :param text: a text
     :return: a list of lower-cased tokens without punctuation
     """
-
+    if isinstance(text, str) == False:
```

Comment:
I repeat that `== False` and `==True` is a very bad practice. 

Basically, conditions operate with boolean values `True` or `False`, and `isinstance`, in this case, already retrieves one. 
Comparison with `==` also returns either `True` or `False`, so checking it in such a way is redundant. 

Please, fix it

Reply:
fixed